src/main/java/org/gridsuite/study/server/service/NetworkService.java

```diff
@@ -96,4 +98,10 @@ UUID getNetworkUuid(Network network) {
     List<VariantInfos> getNetworkVariants(UUID networkUuid) {
         return networkStoreService.getVariantsInfos(networkUuid).stream().sorted(Comparator.comparing(VariantInfos::getNum)).collect(Collectors.toList());
     }
+
+    boolean existeVariant(UUID networkUuid, String variantId) {
```

Comment:
Shouldn't `existeVariant` be instead named `existVariant` ?

Reply:
Renamed